libs/scripts/scripts.js

```diff
@@ -123,7 +123,7 @@ const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
-  imsClientId: 'milo',
+  imsClientId: 'fedsmilo',
```

Comment:
I assume this will not go into the final code!

Reply:
I see this is becoming a pattern used so that QA can perform testing. Should we have some logic that replaces the IMS client id based on a query parameter? Should we add some documentation on some steps to replace it via the browser developer tools? That way we wouldn't have to worry about this update reaching the `main` branch